--- configure.orig	2025-01-20 16:12:31
+++ configure	2025-01-20 16:14:01
@@ -13813,14 +13813,14 @@
 
 
 # check if we need libintl for locale functions
-{ printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for textdomain in -lintl" >&5
-printf %s "checking for textdomain in -lintl... " >&6; }
+{ printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for textdomain in -lgnuintl" >&5
+printf %s "checking for textdomain in -lgnuintl... " >&6; }
 if test ${ac_cv_lib_intl_textdomain+y}
 then :
   printf %s "(cached) " >&6
 else $as_nop
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lintl  $LIBS"
+LIBS="-lgnuintl  $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
@@ -13853,7 +13853,7 @@
 
 printf "%s\n" "#define WITH_LIBINTL 1" >>confdefs.h
 
-        LIBS="-lintl $LIBS"
+        LIBS="-lgnuintl $LIBS"
 fi
 
 
